app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
+end
```

Comment:
- [ ] Good job adding the `message` model, the user can now create a greeting and be able to view it on the back end.
![image](https://user-images.githubusercontent.com/81932478/235937765-155e53cc-6023-4427-b8f4-615794558b07.png)
However, you have to ensure the front-end is able to display the greetings. Currently when I run the front end, no message is being displayed.
![image](https://user-images.githubusercontent.com/81932478/235939084-a2c0c490-5b54-450d-9fbd-c977a76ae3a1.png)

Reply:
Well resolved ✔️ 

![image](https://user-images.githubusercontent.com/38088818/236183902-a857fbd3-23c4-4f97-b32e-a4a96d6fbd7c.png)